app/javascript/styles/mastodon/components.scss

```diff
@@ -2027,6 +2025,32 @@ body > [data-popper-placement] {
       font-size: 15px;
     }
   }
+
+  &__counter {
+    $height: 16px;
+    $h-padding: 5px;
+
+    position: absolute;
+    bottom: 0;
+    right: 0;
+
+    // Using transform here so the counter is slightly outside of the avatar, but does not change the bounding box of the avatar to keep alignment correct
+    transform: translate(3px, 3px);
```

Comment:
Since it's `position: absolute`, does that really change anything? Wouldn't `bottom: -3px; inset-inline-end: -3px` just work?

Reply:
Good point, I forgot that `position: absolute` made the element escape the bounding box.